### ISSUE TITLE
Creating the "First User" must be optional (or configurable)

### ISSUE THREAD

**Issue by kobliha** [agama-project/agama#1418]
Agama will not allow clicking /Install/ before I define the first user, but there are systems that do not need the first user, e.g.., root-only systems that are managed by Salt or similar. Some servers just don't need any other users. If want it to be enforced for TW/Leap, then it needs to be configurable per product as SLES can skip creating that user.

**Comment by ancorgs** [comment]
Yes, that's a topic we need to better define based on the possible distributions and their policies/approaches.

Having said so, it's not true you cannot install before the first user is defined. You cannot install until some authentication has been defined. That authentication can be a first user or can be a password (or SSH key) for root.

**Comment by kobliha** [comment]
![First user not defined](https://github.com/openSUSE/agama/assets/463995/2c78181e-cf1f-4e68-879d-440661779181)

I see, so it's actually a confusing text. After setting the root password I can actually install the system, but Users still complain.

Possible understanding of the text above: If you do not create the user, you will not be able to log into the system.

But that's not really true. You can still log in locally. The text only applies to SSH as root is not permitted to log in with password. Moreover, root can actually login with SSH key.

**Comment by kobliha** [comment]
And on the other hand, if I define the first user, I can install the system. Even without defining the root password or SSH key which makes the system unusable as `sudo` still works the old way, i.e., not the Ubuntu way. So, root password (or SSH key) is still needed for that.

**Comment by dgdavid** [comment]
No longer relevant after https://github.com/agama-project/agama/pull/1973, which grant sudo power to the first user by adding it to wheel group.

Slightly related with https://github.com/agama-project/agama/pull/1999 too.

Thanks for the patience!
